packages/offix-datastore/src/ModelSchema.ts

```diff
@@ -11,6 +11,10 @@ export interface DataSyncProperties extends JSONSchema7 {
   primary?: boolean;
   default?: any;
   encrypted?: boolean;
+  /**
```

Comment:
@Eunovo thanks, yeah I cut a few corners to get the last PR merged. I need to come back and add comments to everything.

Reply:
The `key` param carried over from my code though. So, no problem